test/validation/api/ipsec/ipsec_test_out.c

```diff
@@ -1046,5 +1088,6 @@ odp_testinfo_t ipsec_out_suite[] = {
 				  ipsec_check_esp_null_sha256),
 	ODP_TEST_INFO_CONDITIONAL(test_out_ipv4_udp_esp_null_sha256,
 				  ipsec_check_esp_null_sha256),
+	ODP_TEST_INFO(test_esp_out_in_all),
```

Comment:
I'd prefer having individual test cases as it's easier to control which ones to run (and in which order) while developing an implementation (or debugging a failing test case). Some times a bug (e.g. memory leak) in a previous test case may cause a latter test case to fail.